tests/wpt/mozilla/tests/mozilla/referrer-policy/generic/subresource/a-tag.html

```diff
@@ -3,34 +3,44 @@
   <head>
   </head>
   <body>
-    <a id="button">go</a>
     <script>
       var url = new URL(window.location.toString());
       var id = url.searchParams.get("id");
       var redirection = url.searchParams.get("redirection");
       var cache_destroyer = url.searchParams.get("cache_destroyer");
       var tag_attrs = url.searchParams.get("tagAttrs");
-
-      console.log(id);
-      console.log(redirection);
-      console.log(cache_destroyer);
-      console.log(tag_attrs);
+      var meta_referrer = url.searchParams.get("metaReferrer");
 
       var referrer_policy;
       try {
-        referrer_policy = JSON.parse(tag_attrs);  
+        referrer_policy = JSON.parse(tag_attrs);
       } catch(e) {
         referrer_policy = {};
       }
 
-      var a = document.getElementById("button");
+      if (meta_referrer !== "undefined") {
```

Comment:
We could move all of the dynamic page changes into the python script if we used it (just do string replacement on stuff like `<!-- a tag -->`. Not sure if that's better or not.